application/controllers/admin/dashboards.py

```diff
@@ -277,7 +280,10 @@ def dashboard_create(admin_client, module_types, form):
         admin_client.create_dashboard(dict_for_post)
         flash('Created the {} dashboard'.format(form.slug.data), 'success')
         del session['pending_dashboard']
-        return redirect(url_for('dashboard_list'))
+        if 'save_and_continue' in request.form:
+          return redirect(url_for('dashboard_form'))
```

Comment:
Ah, actually. I think this need to go to `redirect(url_for('dashboard_form', uuid=uuid))` if possible.